docker/test/fasttest/Dockerfile

```diff
@@ -20,12 +20,6 @@ RUN apt-get update \
         zstd \
     --yes --no-install-recommends
 
-# Install CMake 3.20+ for Rust compilation
-RUN apt purge cmake --yes
```

Comment:
Hm, why is cmake no longer build into the fastest dockerfile?

Reply:
It's in the source image `test-utils`